core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -2060,4 +2196,11 @@ object ZIO extends ZIO_R_Any {
   private[zio] final class FiberRefModify[A, B](val fiberRef: FiberRef[A], val f: A => (B, A)) extends UIO[B] {
     override def tag = Tags.FiberRefModify
   }
+  private[zio] final class Trace[R] extends ZIO[R, Nothing, ZTrace] {
+    override def tag = Tags.Trace
+  }
+
+  private[zio] final class TracingStatus[R, E, A](val zio: ZIO[R, E, A], val flag: Boolean) extends ZIO[R, E, A] {
```

Comment:
For completeness, would be good to have a way to check tracing status, like `checkInterrupt`.

Reply:
:+1: